src/page/ConfigPageLayout/tabs/AccessTokensTab.tsx

```diff
@@ -42,8 +42,8 @@ export function AccessTokensTab() {
         documentation to learn more about how to interact with the synthetic monitoring API.
         <Space v={2} />
         <Button
-          tooltip={!canCreateAccessToken ? 'You do not have permission to generate access tokens.' : undefined}
-          disabled={!canCreateAccessToken}
+          tooltip={!canWriteTokens ? 'You do not have permission to generate access tokens.' : undefined}
+          disabled={!canWriteTokens}
```

Comment:
Nit: It would be more helpful to the user to show an alert with a more detailed message.

"To generate an access token, you need to have \<permission>. To add \<permission>, contact an administrator"

Reply:
Good idea. Added the alert:

- without permission
![image](https://github.com/user-attachments/assets/75b27933-4cc0-4805-ba9a-71ba1742a803)

- with permission
![image](https://github.com/user-attachments/assets/97eac690-b055-4671-bea0-ee34addeabc7)